cylc/uiserver/app.py

```diff
@@ -15,11 +15,24 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 """
-Cylc UI Server can be configured using a ``jupyter_config.py`` file located in:
+Cylc UI Server can be configured using a ``jupyter_config.py`` file, loaded
+from a hierarchy of locations. This hierarchy includes the prepackaged
+configuration, the site directory, which defaults to ``/etc/cylc/uiserver`` but
+can be set with the environment variable ``$CYLC_SITE_CONF_PATH`` ) and
```

Comment:
MismatchedBracketError:

```suggestion
configuration, the site directory (which defaults to ``/etc/cylc/uiserver`` but
can be set with the environment variable ``$CYLC_SITE_CONF_PATH``) and
```